SUMMARY.md

```diff
@@ -3,11 +3,11 @@
 * [Introduction](README.md)
 
 ## Read
-* [BOLD-00](BOLD-00.md)
-* [BOLD-01](BOLD-01.md)
-* [BOLD-02](BOLD-02.md)
-* [BOLD-03](BOLD-03.md)
-* [BOLD-04](BOLD-04.md)
+* [BOLD Specifications](00-introduction.md)
```

Comment:
Introduction

Reply:
yes although we already have an "Introduction" link in the left sidebar that points to the readme/front page

Reply:
okay nvm I see the change now thanks